### ISSUE TITLE
Plugin layout is broken with 3.0.2.1

### ISSUE THREAD

**Issue by lekeno** [Marginal/EDMarketConnector#312]
Hi,

It could be something on my end but with 3.0.2.1, my simple label + hypertextlabel UI on the EDMC window has a weird layout (https://github.com/lekeno/edr/issues/130).

Before:
 EDR: [status string]

After:
        EDR:    
  [status string]

My code:
```
    def app_ui(self, parent):
        label = tk.Label(parent, text=u"EDR:")
        self.status_ui = ttkHyperlinkLabel.HyperlinkLabel(parent, textvariable=self._status, anchor=tk.W)
        return (label, self.status_ui)
```

**Comment by Marginal** [comment]
Sorry about that. Fixed in release 3.03.

**Comment by lekeno** [comment]
Thank you!
